package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "metal-metal-key",
```

Comment:
This should be just `metal-key`. Same goes for the repository name in this file.